stdlib/number.gr

```diff
@@ -346,3 +346,56 @@ export let isInfinite = (x: Number) => {
  * @since v0.4.5
  */
 export parseInt
+
+/**
+ * Computes how many times pi has to be subtracted to achieve the required bounds for sin.
+ */
+let reduceToPiBound = (radians: Number) => {
+  let quot = floor(radians / pi)
+  quot
+}
+
+/**
+ * Computes the sine of a number using Chebyshev polynomials. Requires the input to be bounded to (-pi, pi).
+ */
+let chebyshevSine = (radians: Number) => {
```

Comment:
Could you add a comment to this function with a link to information about the algorithm?

Reply:
If the link I added in the newest commit doesn't load fast enough, please try the [archived version](https://web.archive.org/web/20220416080650/mooooo.ooo/chebyshev-sine-approximation/) instead.